### PR TITLE
[Skill Template] Updated with new libs, LG, activity handler, main dialog changes

### PR DESCRIPTION
**Pull request by lauren-mills**
### Purpose
*What is the context of this pull request? Why is it being done?*

### Changes
*Are there any changes that need to be called out as significant or particularly difficult to grasp? (Include illustrative screenshots for context if applicable.)*

### Tests
*Is this covered by existing tests or new ones? If no, why not?*

### Feature Plan
*Are there any remaining steps or dependencies before this issue can be fully resolved? If so, describe and link to any relevant pull requests or issues.*

#### General
- [ ] I have commented my code, particularly in hard-to-understand areas	
- [ ] I have added or updated the appropriate tests	
- [ ] I have updated related documentation

#### Bots
- [ ] I have validated that new and updated responses use appropriate [Speak](https://docs.microsoft.com/en-us/azure/bot-service/dotnet/bot-builder-dotnet-text-to-speech?view=azure-bot-service-3.0) and [InputHint](https://docs.microsoft.com/en-us/azure/bot-service/dotnet/bot-builder-dotnet-add-input-hints?view=azure-bot-service-3.0) properties to ensure a high-quality speech-first experience
- [ ] I have replicated language model changes across the English, French, Italian, German, Spanish, and Chinese `.lu` files and validated that deployment is successful

#### Deployment Scripts
- [ ] I have replicated my changes in the **Virtual Assistant Template** and **Sample** projects
- [ ] I have replicated my changes in the **Skill Template** and **Sample** projects

### CODE REVIEW

**Review by lzc850612 — APPROVED** [review]

File: templates/Skill-Template/csharp/Sample/SkillSample/Responses/MainResponses.lg [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+- No problem, let me know if I can help you with anything else.
+- Sure, I'll cancel your request. Let me know if I can help with anything else
+
+# UnsupportedMessage
+- I'm sorry, I'm not able to help with that.
+- Sorry, that's not something I can help with yet.
+- Apologies, I don't know how to do that right now.
+- Unfortunately I can't help you with that right now. Apologies.
+
+# ConfusedMessage
```
Comment (DingmaomaoBJTU) [inline templates/Skill-Template/csharp/Sample/SkillSample/Responses/MainResponses.lg]: Use activity template to add input hint (expectinginput?)

File: templates/Skill-Template/csharp/Sample/SkillSample/Responses/SampleResponses.lg [hunk 2]
```diff
@@ -0,0 +1,6 @@
+﻿# NamePrompt
+- What is your name?
```
Comment (DingmaomaoBJTU) [inline templates/Skill-Template/csharp/Sample/SkillSample/Responses/SampleResponses.lg]: similar, input hint?

**Review by darrenj — APPROVED** [review]

File: templates/Skill-Template/csharp/Sample/SkillSample/Responses/SampleResponses.lg (hunk 2 above)
Comment (lauren-mills) [reply templates/Skill-Template/csharp/Sample/SkillSample/Responses/SampleResponses.lg]: Done

File: templates/Skill-Template/csharp/Sample/SkillSample/Responses/MainResponses.lg (hunk 1 above)
Comment (lauren-mills) [reply templates/Skill-Template/csharp/Sample/SkillSample/Responses/MainResponses.lg]: Done

**Pull request merged**